mantidproject/mantid
# Inconsistent enginx accuracy performance using damping minimizer

If a fit accuracy result is NaN, the corresponding runtime for it is now also recorded as NaN.

**To test:**

Run benchmarks on Neutron Data using the `Damped GaussNewton` minimizer (this combination produces NaN accuracy results). Ensure that the corresponding runtime results are also recorded as NaN.

Fixes #18480 



---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
